spec/daru/io/importers/xlsx_spec.rb

```diff
@@ -0,0 +1,67 @@
+RSpec.describe Daru::IO::Importers::XLSX do
+  subject { described_class.new(path, sheet: sheet, headers: headers).call }
+
+  let(:sheet)   { 0    }
+  let(:headers) { true }
+
+  context 'parses default sheet' do
```

Comment:
Wouldn't it be logical to parse the 1st sheet by default? The default is set as 0, even for the `from_excel` method.